CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/ViewModels/BaseLineViewModel.cs

```diff
@@ -78,88 +52,158 @@ private void fillLayout()
             ;
             for (int i = m_counterID; i < m_counterID + 3; i++)
```

Comment:
why 3 of this?

Reply:
Just for demonstration, this is going to change following new orders coming in